public/docs/ts/latest/cookbook/a11y.jade

```diff
@@ -0,0 +1,1087 @@
+    structure, you make sure that all users can navigate your pages correctly. Where you place the elements
+    with `CSS` does not affect this order. We call this the `Separation of Content and Presentation`.
+
+:marked
+  We saw in the labeling section that we get a lot of standard accessibility functionality out of the box when we use
+  `native form controls`, and here it is no different. **DO NOT** change the focus order with script unless
+  it is for a very specific functional reason you cannot solve with the default flow, like focusing 
+  on an error message the user needs to see. 
+  
+  Let's have a look at a basic example of separating content from presentation. We have a basic layout based on a list 
+  of countries, asking for two pieces of information per country. It looks like this: 
+
+figure.image-display
+  img(src="/resources/images/cookbooks/a11y/focus-flow-clean.png" alt="Collection of inputs based on country list separated into columns per information type")
+
+.l-sub-section
+  :marked
+    Note how we repeated the country name in both related `input labels`. Remember: **DO NOT** rely on visual context alone
+    when labeling `HTML elements`. A person with a visual disability cannot see this context so you need to tell them, 
+    through the screen reader, what the current element relates to.
+
+:marked
+  If we create this layout in the `HTML DOM Tree`, one column below the other, we end up with the following focus flow:
+
+figure.image-display
+  img(src="/resources/images/cookbooks/a11y/focus-flow-bad.png" alt="Incorrect focus flow grouping taborder into columns")
+
+:marked
+  The example's simplicity and the choice of labeling make this usable, but the flow is illogical. Generally,
+  the user would want to think about one country at a time.
+  
+  A far superior focus flow moves the focus in the order that a reader would encounter it:
+
+figure.image-display
+  img(src="/resources/images/cookbooks/a11y/focus-flow-good.png" alt="Correctly flowing focus by country")
+
+:marked
+  We do this by managing the focus flow through the `content` with HTML alone and changing the visual `presentation` with
+  `CSS`. Just like that we are able to create the required flow with absolutely no scripting! 
+  
+  Here is the HTML:
+
++makeExample('cb-a11y/ts/app/managing-focus/a11y-managing-focus.component.html', 'cb-a11y-managing-focus-flow')
+
+:marked
+  ### Skiplinks
+  
+  For a sighted person who navigates the web using a mouse, it is very easy to skip sections of web pages.
+  This could be because they are returning to a website they know very well, or because they can immediately see and interact
+  with a specific section of interest.
+
+  Similarly, we need to provide quick in-page links to help users who rely on assistive
+  technology to navigate our page.
+
+  These are called `skiplinks` and remain completely hidden until focused through `keyboard navigation`.
```

Comment:
Right, but typically the `code` backticks are for actual snippets of code or commands - variable names, hex values, CLI commands, that sort of thing. Whereas technical terms get the _em treatment_.